challenge_6/python/ajschrier/IntRange.py

```diff
@@ -0,0 +1,41 @@
+ARROW = '->'
+
+
+def outputRange(inputList):
+    # Helper Vars
+    outputList = []
+    rangeStart = None
+    rangeEnd = None
+
+    for i in inputList:
+        # Detect the beginning of the list
+        if rangeStart is None:
+            rangeStart = i
+            rangeEnd = i
+        # Check for invalid sequence
+        elif rangeEnd is not i-1:
```

Comment:
I should add, this differs from the above `if rangeStart is None` check because `None` is an explicit object that you are testing against it's direct identity. All None values are the exact same object.